extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/HibernateOrmRecorder.java

```diff
@@ -156,11 +217,8 @@ public void doValidation(HibernateOrmRuntimeConfig hibernateOrmRuntimeConfig, St
         if (!"none".equals(schemaManagementStrategy)) {
             return;
         }
-        new Thread(new Runnable() {
-            @Override
-            public void run() {
-                SchemaManagementIntegrator.runPostBootValidation(puName);
-            }
-        }, "Hibernate post-boot validation thread for " + puName).start();
+        new Thread(
+                () -> SchemaManagementIntegrator.runPostBootValidation(puName),
+                "Hibernate post-boot validation thread for " + puName).start();
```

Comment:
I've been told repeatedly to avoid lambdas in Quarkus extension code, for weird reasons related to memory/performance.

I won't ask that you avoid them everywhere, but please don't go out of your way to introduce more lambdas. Please revert this particular change? :)